modules/nanomethphase/1.0/nanomethphase.smk

```diff
@@ -0,0 +1,238 @@
+    _nanomethphase_all
+
+
+##### RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _promethion_input:
+    input:
+        bam = CFG["inputs"]["sample_bam"],
+        bai = CFG["inputs"]["sample_bai"],
+        mc = CFG["inputs"]["meth_calls"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam.bai",
+        mc = CFG["dirs"]["inputs"] + "meth_calls/{seq_type}--{genome_build}/{sample_id}.calls.tsv.gz"
+    run:
+        op.absolute_symlink(input.bam, output.bam)
+        op.absolute_symlink(input.bai, output.bai)
+        op.absolute_symlink(input.mc, output.mc)
+
+rule _methyl_call_processor:
+    input:
+        calls = CFG["dirs"]["inputs"] + "meth_calls/{seq_type}--{genome_build}/{sample_id}.calls.tsv.gz"
+    output: 
+        bed = CFG["dirs"]["methyl_call"] + "{seq_type}--{genome_build}/{sample_id}.bed.gz"
+    threads:
+        CFG["threads"]["methyl_call"]
+    conda:
+        CFG["conda_envs"]["nanomethphase"] 
+    resources: 
+        mem_mb = CFG["mem_mb"]["methyl_call"]        
+    log:
+        stderr = CFG["logs"]["methyl_call"] + "{seq_type}--{genome_build}/{sample_id}/methyl_call_processor.stderr.log"             
+    shell:
+        op.as_one_line("""
+        nanomethphase methyl_call_processor -mc {input.calls} --threads {threads} | 
+        sort -k1,1 -k2,2n -k3,3n | bgzip > {output.bed} && tabix -p bed {output.bed}
+        """)   
+
+rule _clair3:
+    input:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        fasta = reference_files("genomes/{genome_build}/genome_fasta/genome.fa")
+    params: 
+        model = CFG["clair3"]["model"],
+        dir = CFG["dirs"]["clair3"] + "{seq_type}--{genome_build}/{sample_id}"
+    threads:
+        CFG["threads"]["clair3"]    
+    conda :
+        CFG["conda_envs"]["clair3"]  
+    resources: 
+        mem_mb = CFG["mem_mb"]["clair3"]     
+    log:
+        stderr = CFG["logs"]["clair3"] + "{seq_type}--{genome_build}/{sample_id}/clair3.stderr.log"    
+    output:
+        vcf = CFG["dirs"]["clair3"] + "{seq_type}--{genome_build}/{sample_id}/phased_merge_output.vcf.gz"       
+    shell:
+         op.as_one_line("""run_clair3.sh  
+            -b {input.bam} -f {input.fasta} -t {threads} -p "ont"
+            -m {params.model} -o {params.dir}
+            --remove_intermediate_dir --enable_phasing """)
+
+rule _filter_clair3:
+    input:
+        vcf = str(rules._clair3.output.vcf)
+    output: 
+        filtered = CFG["dirs"]["filter_clair3"] + "{seq_type}--{genome_build}/{sample_id}.filtered_phased_merged.vcf.gz"
+    log:
+        stderr = CFG["logs"]["filter_clair3"] + "{seq_type}--{genome_build}/{sample_id}/filter_clair3.stderr.log"      
+    shell :
+        "zcat {input.vcf} | awk '$6 > 20' | gzip > {output.filtered}"       
```

Comment:
Is this value of 20 something that could/should be user-configurable via the config?